hack/common.sh

```diff
@@ -122,10 +122,22 @@ run_WMCO() {
       fi
   fi
 
-  # Add the "openshift.io/cluster-monitoring:"true"" label to the operator namespace to enable monitoring
-  if ! oc label ns $WMCO_DEPLOY_NAMESPACE openshift.io/cluster-monitoring=true --overwrite; then
-      return 1
-  fi
+  # required labels for WMCO namespace
+  declare -a NS_LABELS=(
+    # enable monitoring
+    "openshift.io/cluster-monitoring=true"
```

Comment:
Please do not include this change as part of this PR.

Reply:
The `openshift.io/cluster-monitoring=true` was present. This change modularizes the logic for adding the required labels, as mentioned in the commit message.